tests/python/pytorch/graphbolt/gb_test_utils.py

```diff
@@ -9,9 +9,10 @@
 import torch
 
 
-def rand_csc_graph(N, density):
+def rand_csc_graph(N, density, bidirection_edge=True):
     adj = sp.random(N, N, density)
-    adj = adj + adj.T
+    if bidirection_edge:
```

Comment:
Note that even if the bidirection_edge, there still could be bidirection-edge, is it what you want?

Reply:
Yes. I know even if `bidirection_edge == False` there could still be bidirectional edges, I just want `adj = adj + adj.T` to be conditionally executed.